dulwich/reflog.py

```diff
@@ -93,3 +93,62 @@ def read_reflog(f):
     """
     for line in f:
         yield parse_reflog_line(line)
+
+
+def drop_reflog_entry(f, index, rewrite=False):
```

Comment:
This could have a porcelain equivalent to `git reflog delete ...` but I'm not sure it's needed

Reply:
That would be nice, but I don't think that needs to be in this pull request.